MANIFEST

```diff
@@ -0,0 +1,7 @@
+# file GENERATED by distutils, do NOT edit
```

Comment:
this shouldn't need to be checked in; it's auto-generated when building the package, right?